tests/windows_test.go

```diff
@@ -316,49 +238,70 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 			})
 		})
 	})
-
-	Context("[ref_id:142]with kubectl command", func() {
-		var yamlFile string
-		BeforeEach(func() {
-			clientcmd.SkipIfNoCmd("kubectl")
-			yamlFile, err = generateVMIJson(windowsVMI, GinkgoT().TempDir())
-			Expect(err).ToNot(HaveOccurred())
-		})
-
-		It("[test_id:223]should succeed to start a vmi", func() {
-			By("Starting the vmi via kubectl command")
-			_, _, err = clientcmd.RunCommand("kubectl", "create", "-f", yamlFile)
-			Expect(err).ToNot(HaveOccurred())
-
-			tests.WaitForSuccessfulVMIStartWithTimeout(windowsVMI, 360)
-		})
-
```

Comment:
Should we keep testid 239 ? This covers all major functions

Reply:
We should separate the testing of kubectl from testing the windows guest